torch/fx/proxy.py

```diff
@@ -94,6 +94,7 @@ def __exit__(self, *args):
     "recompute",
     "from_node",
     "quantization_tag",
+    "subgraph"
```

Comment:
"subgraph" is a pretty generic-sounding name. What about something more specific, like "is_backward_mutation"?

Reply:
I think we can reuse this for other partitioner related actions. I will rename to "partitioner_tag": "mutation_in_backward"